src/cljs/athens/views/blocks/types/default.cljs

```diff
@@ -371,7 +371,7 @@
 
               (when (= @drag-target :before) [drop-area-indicator/drop-area-indicator {:placement "above"}])
 
-              [editor/editor-component
+              [:f> editor/editor-component
```

Comment:
I wasn't able to add this after merging master in. In master, `editor/editor-component` was rolled back into `blocks.core/block-el`.